src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -430,6 +430,17 @@ private ImmutableArray<DiagnosticId> GetFixableDiagnosticIds(CodeFixProvider fix
             }
         }
 
+        private ImmutableArray<string> GetAndTestFixableDiagnosticIds(CodeFixProvider f)
+        {
+            var ids = f.FixableDiagnosticIds;
+            if (ids.IsDefault)
+            {
+                throw new ArgumentException(WorkspacesResources.FixableDiagnosticIdsIncorrectlyInitialized, nameof(f.FixableDiagnosticIds));
```

Comment:
I agree. I would recommend renaming `f` to `codeFixProvider`, and using `nameof(codeFixProvider)` for the exception.